### PR TITLE
LDAP: quote LDAP debug filter config entry

### PR DESCRIPTION
**Pull request by mccollam**
This value needs to be quoted for the file to parse correctly.  Uncommenting it as is results in "Failed to load LDAP config file: Near line 4 (last key parsed 'log.filters'): expected value but found \"ldap\" instead".

**What this PR does / why we need it**:
Uncommenting the debug config line as-is results in an error parsing the file.  Quoting it causes debugging to be enabled correctly.

### CODE REVIEW

**Comment by aknuds1** [comment]
Do you mean to uncomment the following in ldap.toml?

```
# [log]
# filters = ldap:debug
```
AFAICT, you're supposed to copy it into grafana.ini (or rather custom.ini?).

**Comment by mccollam** [comment]
No, I don't mean to uncomment it.  Uncommenting the line as it is currently written will not work -- it throws an error rather than enabling logs.

Adding quotes around "ldap:debug" causes the line to work correctly when uncommented.  (I'm leaving it commented intentionally as the intent is not to make the *default* behavior to emit debug logs.)

You can verify this by:

1) Uncommenting the line as is (without quotes)
2) Observe that you get an error in the logs rather than LDAP debug logs
3) Adding quotes around "ldap:debug"
4) Observe that you no longer get an error in the logs but do in fact get LDAP debug logs

Basically the intent of this PR is to make it so that someone can uncomment the line and get the expected behavior.

**Comment by aknuds1** [comment]
To clarify for eventual other readers, @mccollam and I discussed it, and we should be on the same page that those two lines are supposed to be copied into the main Grafana configuration file instead.
